ui/pages/onboarding-flow/metametrics/metametrics.test.js

```diff
@@ -133,6 +134,7 @@ describe('Onboarding Metametrics Component', () => {
   });
 
   it('should render the Onboarding component when the current date is after the new privacy policy date', () => {
+    delete process.env.IN_TEST;
```

Comment:
This pattern is copied from an existing test.
https://github.com/MetaMask/metamask-extension/blob/develop/shared/modules/i18n.test.ts#L94